.github/workflows/nightly.yaml

```diff
@@ -17,3 +17,48 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           args: --namespace=test-clusters --image=cilium/cilium-test-dev:${{ steps.vars.outputs.sha_short }} "/usr/local/bin/cilium-test-gke.sh" "docker.io/cilium/cilium:latest" "docker.io/cilium/operator-generic:latest" "docker.io/cilium/hubble-relay:latest" "NightlyPolicyStress"
+  baseline-test:
+    name: Start performance test
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Trim git sha
+        id: vars
+        run: echo "::set-output name=sha_short::$(echo ${GITHUB_SHA:0:9})"
```

Comment:
is `sha_short` used anywhere in this job ?

Reply:
Nope, it was initially, but then usage got removed. Will delete.